test/e2e/app/app.go

```diff
@@ -129,8 +141,9 @@ func DefaultConfig(dir string) *Config {
 // 2 - blob is empty ([]byte{})
 // 3 - blob is variable length string that contains "BLOBXXX" where XXX is height multiplied by 0x80 in hex
 // 4 - blob is the size of MaxBlobSizeBytes and contains height truncated into two bytes repeated.
-func blobOracle(height int64) ([]byte, bool) {
-	switch height % 4 {
+func blobOracle(height int64, blobMaxBytesUpdateHeight int64) ([]byte, bool) {
```

Comment:
The parameter added in this PR is no longer used in the current version, is that intentional?

Reply:
Yes, because it is not needed anymore, will remove it. `blobOracle` is not called if the blobs are not enabled.